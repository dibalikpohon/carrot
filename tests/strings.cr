-- check equality of string
one: str = "Hello "

two: str = "World!"
three: str = "World!"

println("one and two is the same? ", one == two)

println("two and three is the same? ",three == two)

println("one and two is not the same? ", one != two)

-- adding between two strings
four: str = one + two
println(four)
-- print one and two to check if it still
-- accessible and don't cause and segfaults
println(one)
println(two)
println(four == "Hello World!")
