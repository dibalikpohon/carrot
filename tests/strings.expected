one and two is the same? false
two and three is the same? true
one and two is not the same? true
Hello World!
true
